src/gam.py

```diff
@@ -503,6 +503,34 @@ def getCharSet(i):
     return (i, GC_Values.get(GC_CHARSET, GM_Globals[GM_SYS_ENCODING]))
   return (i+2, sys.argv[i+1])
 
+def cleanCourseId(courseId):
```

Comment:
cleanCourseId implies courses can be dirty. Rename to something more meaningful.